### PR TITLE
documentation/finetuning-style-guide

### PR DESCRIPTION
**Pull request by ghost**
documentation/finetuning-style-guide

### CODE REVIEW

File: docs/development/contribution-documentation/documentation-style-guide/README.md [hunk 1]
```diff
@@ -1,10 +1,4 @@
----
-sidebar_navigation:
-  title: Documentation style guide
-  priority: 700
-description: What styles and formats to follow when contributing to the OpenProject documentation
-keywords: documentation style guide, style guide, format, style
----
+
```
Comment (as-op) [inline docs/development/contribution-documentation/documentation-style-guide/README.md]: The yaml header has been accidentally removed.

**Review by as-op — CHANGES_REQUESTED** [review]

File: docs/development/contribution-documentation/documentation-style-guide/README.md (hunk 1 above)
Comment (ghost) [reply docs/development/contribution-documentation/documentation-style-guide/README.md]: Hi @as-op
did you fix that already? I think Typora default behaviour.
I do not see this written in the style guide, maybe somewhere else, we could raise the awareness for that at the documentation contributor...

Comment (as-op) [reply docs/development/contribution-documentation/documentation-style-guide/README.md]: No, unfortunately I can't fix it here, it's in your fork.

Maybe it's a setting in Typora? So far this did not happen for other Typora users.

**Comment by ghost** [comment]
@as-op  i added the lines that seem to be removed in each commit i do, could you please check? in the next PR i will try to be aware, thx

**Review by as-op — CHANGES_REQUESTED** [review]
Hi @adam-op,

There are now empty lines above the yaml header (which leads to the header being interpreted as markdown). Please remove them again.

Secondly there are now dashes in front of each line. Please remove them again.

**Comment by ghost** [comment]
@as-op i tried again, is it now okay?

File: docs/development/contribution-documentation/documentation-style-guide/README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
----
+```
```
Comment (as-op) [inline docs/development/contribution-documentation/documentation-style-guide/README.md]: One more issue. 

I'm sorry, I oversaw that the header markings have been changed in the markdown, too.
The yaml header needs to be enclosed by three dashes --- not ```

**Review by as-op — CHANGES_REQUESTED** [review]

File: docs/development/contribution-documentation/documentation-style-guide/README.md (hunk 2 above)
Comment (ghost) [reply docs/development/contribution-documentation/documentation-style-guide/README.md]: hi @as-op 
okay, cool, i did that now, please check, thank you

**Review by as-op — APPROVED** [review]

**Pull request merged**